src/components/DataInput/DataInput.cy.tsx

```diff
@@ -13,15 +13,18 @@ import {
 import { hexToRgb, SelectChangeEvent } from "@mui/material";
 import CheckboxGroupInput from "@/components/DataInput/CheckboxGroupInput";
 import PasswordInput from "@/components/DataInput/PasswordInput";
-import { logInfoReturnLogId } from "@/logger/logger";
+
+const mockIDLogger: (logId: string) => void = (logId: string) => {
+    return console.log(logId);
+};
```

Comment:
because console.log is already a void function we can write this a little bit more consicesly as
`const mockIDLogger: (logId: string) => void = console.log`

Reply:
this is now fixed:)